.github/workflows/mypy.yml

```diff
@@ -29,7 +29,12 @@ jobs:
         run : |
           python3 -m pip install types-croniter types-requests types-PyYAML
 
-      - name: mypy
+      - name: mypy sdk
         working-directory: sdk
         run: |
           mypy -m aqueduct --ignore-missing-imports --strict
+      
+      - name: mypy executor
+        working-directory: src/python
+        run: |
+          mypy aqueduct_executor --ignore-missing-imports --exclude tests
```

Comment:
Why is the flags here different from the one above? Also should we add mypy check to the integration tests directory as well? @likawind

Reply:
We should do integration tests but I would prefer to do in a separate PR, since there are probably some additional fix involved.

@kenxu95 do you know how the commands are different? I couldn't properly run run using `-m aqueduct_executor` locally, it only check one file instead of the entire tree

Reply:
I don't know exactly why they're different. I also saw the same issue with `-m`, but I found this command which seemed to give sane results. I wonder if `-m` only looks at the top-level directory?

Reply:
@cw75 did you see any difference running w wo `-m` for SDK?

Reply:
I'm looking into the right flag to use now. Pretty sure at this point `-m` only looks at the python files at the top level directory.

Reply:
@likawind @kenxu95  I think running `mypy .` recursively checks all the Python files. We shouldn't use the `-m` flag.

Reply:
@likawind after some investigation with @kenxu95 , we think we should add:
1. "--ignore-missing-imports"
2. "--strict"
3. "--exclude" if necessary
4. "--implicit-reexport"
flags to all mypy commands.